specification/hanaonazure/resource-manager/Microsoft.HanaOnAzure/2017-06-15-preview/hanaonazure.json

```diff
@@ -0,0 +1,380 @@
+                },
+                "tags": {
+                    "type": "object",
+                    "additionalProperties": {
+                        "type": "string"
+                    },
+                    "description": "Resource tags"
+                }
+            },
+            "x-ms-azure-resource": true
+        },
+        "HanaInstancesListResult": {
+            "properties": {
+                "value": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/HanaInstance"
+                    },
+                    "description": "The list of SAP HANA on Azure instances."
+                },
+                "nextLink": {
+                    "type": "string",
+                    "description": "The URL to get the next set of HANA instances."
+                }
+            },
+            "description": "The response from the List HANA Instances operation."
+        },
+        "HanaInstance": {
+            "properties": {
+                "properties": {
+                    "x-ms-client-flatten": true,
+                    "$ref": "#/definitions/HanaInstanceProperties",
+                    "description": "HANA instance properties"
+                }
+            },
+            "allOf": [
+                {
+                    "$ref": "#/definitions/Resource"
+                }
+            ],
+            "description": "HANA instance info on Azure (ARM properties and HANA properties)"
+        },
+        "HanaInstanceProperties": {
+            "properties": {
+                "hardwareType" : {
+                    "type": "string",
+                    "enum": [
+                        "Cisco_UCS"
+                    ],
+                    "x-ms-enum": {
+                        "name": "HanaHardwareTypeNamesEnum",
+                        "modelAsString": true
+                    },
+                    "description": "Name of the hardware type (vendor and/or thrie product name)"
+                },
+                "hanaInstanceGenericProperties": {
+                    "type": "object",
+                    "description": "Properties of an SAP HANA on Azure instance. Currently an arbitrary JSON object because the RP is currently in active design and development."
+                }
+            },
+            "description": "HANA instance properties"
+        },
+        "OperationList": {
+            "type": "object",
+            "properties": {
+                "value": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/Operation"
+                    },
+                    "description": "List of HANA operations"
+                }
+            },
+            "description": "List of HANA operations"
+        },
+        "Operation": {
+            "type": "object",
+            "properties": {
+                "name": {
+                    "type": "string",
```

Comment:
can those properties (and hence those of `Display`) be marked as readOnly?

Reply:
Added

Reply:
Yes, I've added them. Thanks!